frontend/src/pages/home.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+class NewHomePage extends React.Component {
+  render() {
+    const handleButtonClick = (buttonName) => {
+      window.location.href = `/${buttonName.toLowerCase()}`
+    }
+    const containerStyle = {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: '100vh',
+      fontFamily: 'Roboto, sans-serif',
+    }
+    const buttonStyle = {
+      backgroundColor: '#FFFFFF',
+      border: '2px solid #000000',
+      borderRadius: '10px',
+      color: '#333',
+      textAlign: 'center',
+      textDecoration: 'none',
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      fontSize: '20px',
+      margin: '4px 2px',
+      padding: '5px',
+      cursor: 'pointer',
+      width: '50%',
+    }
+    const imageStyle = {
+      padding: '10px',
+      verticalAlign: 'middle',
+      display: 'block',
+    }
+
+    return (
+      <>
+        <div style={containerStyle}>
+          <p style={{ margin: '0px', fontWeight: 'bold' }}>Welcome to the</p>
+          <h1 style={{ margin: '10px 0px 50px 0px' }}>NICE Network</h1>
+
+          {/* Sign Up */}
+          <button
+            style={buttonStyle}
+            onClick={() => handleButtonClick('sign-up')}
+            onMouseOver={(e) => (e.target.style.borderColor = 'red')}
+            onMouseOut={(e) => (e.target.style.borderColor = '#000000')}
+          >
+            <img
+              src="images/sign_up.png"
+              alt="Button Image Sign Up"
+              style={imageStyle}
+            />
+            Sign Up
+          </button>
```

Comment:
could make the 3 buttons out of a single component if you wanted, pass in the handleButtonClick string and the img src and alt

Reply:
like a new Button component that takes some props, and return something like 3 of these
```
      <Button
        route="sign-up"
        src="images/sign_up.png"
        alt="Button Image Sign Up"
      >
        Sign Up
      </Button>
```